lib/hammer_cli_foreman/smart_class_parameter.rb

```diff
@@ -180,10 +187,15 @@ class RemoveOverrideValueCommand < HammerCLIForeman::DeleteCommand
       end
     end
 
+    HammerCLIForeman::SmartClassParameter.lazy_subcommand('remove-override-value', _("Remove an override value for a specific smart variable"),
+      'HammerCLIForeman::SmartClassParameter', 'hammer_cli_foreman/smart_class_parameter',
```

Comment:
Ditto, we need `HammerCLIForeman::SmartClassParameter::RemoveMatcherCommand` here